packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -2874,18 +2892,26 @@ describe("Runtime", () => {
 			it("should log out of order signal in between signal latency events", () => {
 				// Send 1st signal and process it to prime the system
 				sendSignals(1);
+
 				processSubmittedSignals(1);
+
 				// Send 150 signals and temporarily lose 1
-				sendSignals(150); //            150 outstanding including 1 tracked signal (#101); max #151
-				processSubmittedSignals(95); //  55 outstanding including 1 tracked signal (#101)
-				dropSignals(1); //               54 outstanding including 1 tracked signal (#101)
-				processSubmittedSignals(14); //  40 outstanding; none tracked
-				processDroppedSignals(1); //     40 outstanding; none tracked *out of order signal*
-				processSubmittedSignals(40); //   0 outstanding; none tracked
+				sendSignals(150); // 150 outstanding including 1 tracked signal (#101); max #151
+
+				processSubmittedSignals(95); // 55 outstanding including 1 tracked signal (#101)
+
+				dropSignals(1); // 54 outstanding including 1 tracked signal (#101)
+
+				processSubmittedSignals(14); // 40 outstanding; none tracked
+
+				processDroppedSignals(1); // 40 outstanding; none tracked *out of order signal*
+
+				processSubmittedSignals(40); // 0 outstanding; none tracked
 
 				// Send 60 signals including tracked signal
-				sendSignals(60); //              60 outstanding including 1 tracked signal (#201); max #211
-				processSubmittedSignals(60); //    0 outstanding; none tracked
+				sendSignals(60); // 60 outstanding including 1 tracked signal (#201); max #211
+
+				processSubmittedSignals(60); // 0 outstanding; none tracked
```

Comment:
Previous formatting was much easier to read and follow in my opinion. Better grouping and the counts aligned so you could see where they changed.